### DISCUSSION TITLE
Save slot doesn't load Dk64, screen black.

### DISCUSSION THREAD

**Discussion by Griffith2** [mupen64plus-ae/mupen64plus-ae#1030] [Q&A]
Every time I finish playing I save an slot and It was working normally. Today I tried to load my slot but the screen got black, I tried to reset the game, change some graphics, plugins and still not working, I lost all my progress :(  I hope you may an solution, thank you!

**Comment by fzurita** [comment]
That is as known issue with that game. It's much safer to use in game saves and not use save slots at all with that game. Also, using save slots can unfortunately break in game saves as well which can make it so that you can't save at all.
